docs/docs/integrations/platforms/google.mdx

```diff
@@ -176,7 +176,7 @@ Install the python package:
 pip install langchain-google-memorystore-redis
 ```
 
-See [usage example](docs/docs/integrations/vectorstores/google_memorystore_redis).
+See [usage example](docs/integrations/vectorstores/google_memorystore_redis).
```

Comment:
I'm still seeing some of these links as broken on the vercel preview


https://vercel.live/open-feedback/langchain-git-fork-averikitsch-google-provider-langchain.vercel.app?via=pr-comment-visit-preview-link&passThrough=1

Reply:
done. Thanks!